libfuzzer-dotnet-windows.cc

```diff
@@ -134,7 +137,11 @@ FUZZ_EXPORT int __cdecl LLVMFuzzerInitialize(int *argc, char ***argv)
         die_sys("CreatePipe() failed");
     }
 
-    TCHAR sharedMemName[] = TEXT("LIBFUZZER_DOTNET_SHMEM");
+    UUID uuid;
+    UuidCreate(&uuid);
```

Comment:
Both `UuidCreate` and `UuidToString` can fail. Can you check their return status and exit the program using the `die` function if that status is not `RPC_S_OK`?